modules/instruments/README.md

```diff
@@ -22,7 +22,7 @@ It also does not manage surveys in any way.
 
 ## Permissions
 
-None. Individual instruments may provide their own permissions.
+Accessing instruments requires the `data_entry` permission. Individual instruments may provide their own permissions.
```

Comment:
Does "provide their own permissions" mean it overrides the data_entry or augments it? It looks like it's "permissions they require in addition to" from the code below but that should be clearly in the documentation as well.

Reply:
I'm not sure about the interaction between this hasAccess call and the one in here https://github.com/aces/Loris/blob/826b0a3455eccad59839aa916ca78f4667bdbca5/php/libraries/NDB_BVL_Instrument.class.inc#L318

I would guess to get access to an instrument you would need both to return true ? so it would be augments it.

if you can confirm that I'll try to reformulate

Reply:
It looks like the code you're changing is adding it to the hasAccess of the module class, so I'm pretty sure it augments it and the instrument classes themselves can't override it the way it's written.

But I'm not sure if that's your intention or not based on the documentation change's ambiguity.

Reply:
my intention was to make data_entry ALWAYS required and whatever projects want to add is their call. so I think it works the way it is.

I'll try to change the wording a bit

Reply:
Yeah, that's the way this code looks like it works. The module class is going to return a permission denied before it even tries to load the instrument class.